src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_panorama.py

```diff
@@ -711,7 +760,10 @@ def __call__(
                         callback(i, t, latents)
 
         if not output_type == "latent":
-            image = self.vae.decode(latents / self.vae.config.scaling_factor, return_dict=False)[0]
+            if circular_padding:
+                image = self.decode_latents_with_padding(latents)
```

Comment:
Works for me for now. Ideally, I would have loved to move out the VAE decode part from `decode_latents_with_padding()` to unify this part of the workflow a bit. But okay for now.